modules/services/xsettingsd.nix

```
@@ -21,29 +33,68 @@ in {
           Package containing the <command>xsettingsd</command> program.
         '';
       };
-    };
-  };
 
-  config = mkIf cfg.enable {
-    assertions = [
-      (lib.hm.assertions.assertPlatform "services.xsettingsd" pkgs
-        lib.platforms.linux)
-    ];
-
-    systemd.user.services.xsettingsd = {
-      Unit = {
-        Description = "xsettingsd";
-        After = [ "graphical-session-pre.target" ];
-        PartOf = [ "graphical-session.target" ];
+      config = mkOption {
```

Should be
```suggestion
      settings = mkOption {
```
according to [RFC 42](https://github.com/NixOS/rfcs/blob/master/rfcs/0042-config-option.md#part-1-structural-settings-instead-of-stringly-extraconfig).

---

Thank you, i wasn't sure which one to use.